test/service/test_response.py

```diff
@@ -1251,8 +1248,33 @@ class TestResponseSummary(WebServiceTestCase):
     maxDiff = None
     bundles = WebServiceTestCase.bundles + [
         ('dcccb551-4766-4210-966c-f9ee25d19190', '2018-10-18T204655.866661Z'),
+        ('94f2ba52-30c8-4de0-a78e-f95a3f8deb9c', '2019-04-03T103426.471000Z')  # an imaging bundle
     ]
 
+    def test_summary_response(self):
+        """
+        Verify the /repository/summary response with the following indexed bundles:
+        - bundle=aaa96233, fileCount=2, donorCount=1, totalCellCount=1.0, organType=pancreas
+        - bundle=dcccb551, fileCount=19, donorCount=4, totalCellCount=6210.0, organType=Brain
+        - bundle=94f2ba52, fileCount=227, donorCount=1, totalCellCount=0, organType=brain
```

Comment:
Mention that this is an imaging bundle without cell suspensions.

Reply:
Expanded description of test case in comment